site/content/docs/master/snapshot.md

```diff
@@ -28,8 +28,6 @@ The results of plugins can be inspected without being extracted. By default, it
 sonobuoy results $output [--plugin <name>] [--mode report|detailed|dump]
```

Comment:
Curious:
In this command example it looks like  `--plugin` and `mode` are optional.
If a plugin is not provided, does it assume the `e2e` by default ?

Reply:
No, it prints the info for each plugin if not specified.

So if you try `sonobuoy results tarball` on a default sonobuoy run, you'll see a little summary of e2e and the systemd-logs.